bedrock/mozorg/templates/mozorg/about/includes/m24/hero.html

```diff
@@ -4,17 +4,89 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
+{% from "macros-protocol.html" import picto with context %}
+
 <section class="m24-c-hero">
   <div class="m24-c-copy">
-    <h1 class="m24-c-copy-title m24-t-2xl">Activists for change on and off the screen</h1>
-    <p>We don't answer to shareholders because we don't have any. We're guided by ten principles. And a love for the interweb.</p>
-    <p class="m24-c-copy-cta"><a href="{{ url('mozorg.about.manifesto') }}" class="m24-c-cta" data-cta-text="Mission">Read our mission</a></p>
+    <div class="m24-c-copy-wrapper">
+      <h1 class="m24-c-copy-title m24-t-2xl">Reclaim the internet with us</h1>
+      <p>Mozilla is working to put control of the internet back in the hands of the people using it.</p>
+    </div>
   </div>
 
   <div class="m24-c-hero-media">
-    <picture>
-      <source srcset="{{ static('img/m24/todo-2-1.svg') }}" media="(max-width: 768px)" width="768" height="384"> <!-- 2:1 -->
-      <img src="{{ static('img/m24/todo-27-1.svg') }}" alt="" width="1440" height="533"> <!-- 2.7:1 -->
-    </picture>
+    <div class="m24-c-hero-media-container">
+      <img src="{{ static('img/m24/about/about-hero.jpeg') }}" srcset="/media/img/m24/about/about-hero-high-res.jpeg 2x" alt="" width="1024" height="682">
```

Comment:
**Suggestion:** this is a particularly large image, so I'd recommend serving a responsive image (using `resp_img()`) https://bedrock.readthedocs.io/en/latest/coding.html#resp-img

If this isn't a final image then we can probably merge with this as-is, but we should file followup issues to properly optimize things later on.

Reply:
Actually, there's another issue here in that your `srcset` value isn't using `static()` to build the path. Perhaps that's another reason to use `resp_img()` even with what you have now, since it takes care of this boilerplate for you?